src/main/java/ch/njol/skript/conditions/CondHasBuildRestrictions.java

```diff
@@ -0,0 +1,93 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.doc.*;
```

Comment:
skript doesn't use star imports, you'll probably have to change your intellij settings for wildcard imports.